shared/constants/index.ts

```diff
@@ -13,5 +13,8 @@ export const INTERNAL_OFFERS = [SPRUNGBRETT_OFFER_ALIAS, MALTE_HELP_FORM_OFFER_A
 export const getChatName = (appName: string): string => `Frag${appName} (beta)`
 
 export type Rating = 'positive' | 'negative'
+
+export type SendingStatusType = 'idle' | 'sending' | 'failed' | 'successful'
+export type ExtendedSendingStatusType = SendingStatusType | 'invalidEmail'
```

Comment:
should be also adjusted in native, should I do this in this PR or better create another for native? 🤔

Reply:
I actually think that we should not add custom error messages as status type. If we actually want to show a snackbar for invalid emails, it should be set separately.
```suggestion
```